doc/releases/release-notes-3.4.rst

```diff
@@ -355,6 +355,9 @@ Trusted Firmware-M
 
 * Timer
 
+  * Support added for stopping Nordic nRF RTC timer clock source, which fixes
```

Comment:
This "clock source" mentioned here and in the commit message might be a bit confusing as the LFCLK that drives the RTC is actually not stopped. Perhaps "system timer" would be more suitable.

Reply:
Changed text